src/index.html

```diff
@@ -6,8 +6,37 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+       src="images/imac.jpeg"
+       alt="APPLE A1419 IMac 27"
+       class="card__image">
```

Comment:
```suggestion
      <img
         src="images/imac.jpeg"
         alt="APPLE A1419 IMac 27"
         class="card__image"
       >
       
```
Please pay attention to your code style, check yourself with the [guide](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)